Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -42,6 +42,43 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
                 .to(throwError(ServerError.commandError(code: 59, message: "", errorLabels: nil)))
     }
 
+    func testDropDatabase() throws {
+        let center = NotificationCenter.default
+
+        let client = try MongoClient(options: ClientOptions(eventMonitoring: true))
+        client.enableMonitoring(forEvents: .commandMonitoring)
+
+        var db = client.db(type(of: self).testDatabase)
+        var writeConcern = try WriteConcern(journal: true, w: .number(1))
+
+        let collection = db.collection("collection")
+        try collection.insertOne(["test": "blahblah"])
+
+        var expectedWriteConcerns: [Document] = [
+                                                    ["w": Int32(1), "j": true],
+                                                    ["w": Int32(1), "j": true, "wtimeout": Int32(123)]
+                                                ]
+        let observer = center.addObserver(forName: nil, object: nil, queue: nil) { notif in
```

Comment:
similarly to above, I suggest somehow tracking that we actually saw the `CommandStartedEvent`s happen.
(in this case you could just check that `expectedWriteConcern`s is empty at the end of the test, meaning we saw two events)